hana3d/src/ui/callbacks/asset_bar.py

```diff
@@ -170,6 +172,15 @@ def draw_tooltip(   # noqa: WPS211
         text_revision = f'Modified: {revision_date}'
         bgl_helper.draw_text(text_revision, x_revision, y_revision, font_height, tcol)
 
+    # if it has more than one sku, should add line height foreach one
+
+    y_sku = y_created
+    x_sku = x_created
+    for sku_item in sku.keys():
+        y_sku += line_height
+        text_sku = f'{sku[sku_item]} : {sku_item}'
```

Comment:
Talvez ficasse melhor iterar nos items:
```
for key, value in sku.items()
```

Mas totalmente arbitrário isso, pode deixar assim se fizer mais sentido para você.

Reply:
alterei para 
`for sku_id, library in sku.items():`
já que 'value' gera erro WPS110